storage/smiles/src/main/java/org/openscience/cdk/smiles/SmiFlavor.java

```diff
@@ -172,6 +172,13 @@ private SmiFlavor() {
      */
     public static final int Absolute            = Canonical | Isomeric;
 
+
+    /**
+     * Default SMILES output write Stereochemistry, Atomic Mass, and CXSMILES layers. The
+     * ordering is not canonical.
+     */
+    public static final int Default             = Stereo | AtomicMass | CxSmiles;
```

Comment:
The default constructor is actually deprecated... telling the user to choose what you want. But CXSMILES is backwards compatible with SMILES. Toolkits that support it (ChemAxon,CDK,Indigo) can exchange more information, those that don't (RDKit,OpenBabel,etc) simply ignore. There is information loss, but for our purposes it allows better round tripping when serialising to SMILES.